### DISCUSSION TITLE
Went to buy the full version, but... no?

### DISCUSSION THREAD

**Discussion by Stokestack** [usebruno/bruno#3486] [General]
I'm just getting started with Bruno, and I'm finding it useful and wanted to reward the developers with some dough. In several places I see a $19 purchase option mentioned, but this is nowhere to be found.

No dice anymore?

**Comment by bjartenilsen** [comment]
Pretty sure I saw the $19 personal option on pricing page just one week back.

**Comment by NiklasBr** [comment]
Yeah, strange. I bought a $19 "Golden edition" one-time purchase some months ago, it's gone now it seems.

Edit: I have this URL https://www.usebruno.com/buy-golden-edition in my browser history, but it is a 404 now

[Youtube: Bruno Golden Edition Launch](https://www.youtube.com/watch?v=R3F6Gu7tG8M)

**Reply by theoriginalstove** [reply]
I just purchased the Golden edition a few days ago, but it looks like they just changed their pricing model to a subscription based one: [https://www.usebruno.com/pricing](https://www.usebruno.com/pricing).

**Reply by bjartenilsen** [reply]
Even the Readme.md page on Github still say : "You can buy the [Golden Edition](https://www.usebruno.com/pricing) for a one-time payment of $19!"  - does not seem very well planned, or transparent to change the pricing policy over-night.  A move in wrong direction on both accounts.  I'm just tired of having all these software subscriptions I need to monitor and follow up.

**Comment by thgla** [comment]
I understand that building a product costs money, but why does it have to be a subscription? All I want is a plain and simple API testing tool that lets me store the collection of requests in my source control. Add to that environments, authentication support and I am happy. Just let me buy a perpetual license, and if new features seem enticing I will upgrade. I am very tired of everything being a subscription now.

**Reply by artemudovyk** [reply]
Free version covers all the bases for this, isn't it?

**Reply by thgla** [reply]
Fortunately yes, just checked the license, and since it's MIT licensed I can use it.

**Comment by Stokestack** [comment]
One might argue that if you want to support the project, just subscribe for a few months. I'm not excited about managing this, but it's their business model to do with what they please.

What happens after the subscription? Does the software check for licensing when run?

**Reply by bjartenilsen** [reply]
Sure it's their buisiness model - but when they change this "over night" - without a plan, without updating all pages on this. Still advertise this as $19 for gold edition in this very repo - it doesn't reflect well on the product. 

What would be next they do. Perhaps remove scripting from open source edition?  While the page still say scripting is included in free version.. ?

https://github.com/usebruno/bruno/discussions/3491

Also, as a user, I am entitled to let them know what I feel about their sudden changes.

**Reply by Its-treason** [reply]
> Does the software check for licensing when run?

For the Golden edition: Bruno will check on every start if the license is valid via calling `license.bruno.com`. If you are offline the license will stay active for 14 days and has to be reactivated after that time.

**Reply by NiklasBr** [reply]
> One might argue that if you want to support the project, just subscribe for a few months.

I'm willing to support the project for a fixed sum paid once for every major version or so (this was my expectation when I bought the Golden Edition), maybe in the range of about $20-$50 or so. On top of that I am willing to provide detailed bug reports _and_ pull requests to fix issues where I am knowledgeable enough to do so.

I have done all of the above, bought a license, advocated for the app among colleagues, reported issues, and submitted pull requests to be merged.

Now that the equation has suddenly changed my willingness to support the project has changed.

Bruno & Co are **of course** 100% within their right to do such a change. I am not here to "forbid" them to do such a thing, just to say I am disappointed and ask them to please reconsider.

**Reply by bjartenilsen** [reply]
> For the Golden edition: Bruno will check on every start if the license is valid via calling `license.bruno.com`. If you are offline the license will stay active for 14 days and has to be reactivated after that time.

"Bruno is offline-only." - again from readme file of this repo.

**Reply by Stokestack** [reply]
I'm by no means endorsing this change. Just acknowledging that if they want to risk pissing people off with a rental arrangement, all we can do is let them know that they're losing income. I was ready to pay up, and then found that they don't want to take my money on the terms published in at least a couple of places.

If it phones home, that's a definite nope.

**Reply by drpixel8** [reply]
I don't think you can just subscribe for a month can you?  The pricing page says "Billed Yearly".  So their business model is to push users to the open-source edition, which is free, or to force you to buy an annual subscription at $72 which essentially gets you CSV data driven testing.  I feel like that's a big "no thanks" for most people, but time and sales will tell, I guess.

I think this rollout should be a case study on what not to do.  An email blast to existing licensees would have been nice as a heads up.  The website pricing page should have been updated with an explanation as to the fact that things have changed and the Golden Edition is no longer being offered.  The README, manifesto and similar postings should be updated to reflect that Bruno is indeed a for-profit company and subscription based.

**Reply by drpixel8** [reply]
> > "Bruno is offline-only." - again from readme file of this repo.

Sort of makes them hard to trust.  I've recommended Bruno to many people and I as well as many others have invested a lot of time in writing test cases and scripts against our APIs using Bruno.  Until everything has been clarified, I'm not recommending or using this product any longer.  I don't want to have to rewrite my tests for another API testing tool if Bruno keeps heading in a direction I don't agree with long term.

**Comment by bjartenilsen** [comment]
Also, "We don't want to sell monthly recurring subscriptions" - https://github.com/usebruno/bruno/discussions/269. Sure...

**Reply by kimdre** [reply]
[Enshittification](https://en.wikipedia.org/wiki/Enshittification) is the correct term to describe what the Bruno devs are up to.

**Reply by anhofmann** [reply]
To give another perspective on this: I'm currently investigating if I can tell my company, that we should switch to bruno. It takes an hour to check if bruno has all the features, we need. Then I search for signs if the people behind bruno are paid well. Because if they aren't, I can't use their product for the next 10 years. This isn't enshittification. It could get to that point, but right now, we can't know the future.

**Reply by kimdre** [reply]
Fair point, but we also were at that point already with Postman and we all know how that went.

**Reply by anhofmann** [reply]
> Fair point, but we also were at that point already with Postman and we all know how that went.

In Germany, we have the nice word "Unschuldsvermutung", which translates to 'presumption of innocence'. What I mean: we know that the people who are steering Postman are a bunch of unfathomably greedy a-holes. And we see enshittification in a lot of places. But this doesn't mean that the people behind bruno will do the same. We can't know yet, time will tell. 

Right now, I think it is better that the people behind bruno try to make a sustainable living out of their product instead of just burning out and leave a dead open-source project. I saw enough of that in my lifetime and had a lot of work switching from one dead project, to something else.

**Comment by helloanoop** [comment]
We have posted a detailed update on the changes here: https://blog.usebruno.com/exploring-apis-with-bruno

**Reply by anhofmann** [reply]
One thing that you need to be aware of: it is still easy to find the term "golden edition" and the old price on the bruno pages, github, documentation etc. People will read this and be dissapointed. And as you can see, a lot of them will think that bruno will do the same ultra frustrating thing that postman, insomnia et al. did in the last years. 

You need to earn trust and your audience (developers) are very, veeeery sensitive on that topic. As soon as a small, but loud part of your audience stop trusting you, your product might be dead, because they will use every chance to proclaim "don't trust them, they are the same shit as everyone else". I've seen this over and over, and it makes me sad, because it kills good products. I want products like bruno to thrive, and I know that this needs money, a lot of it. A lot of users are still not really aware of this fact, and this makes things like price changes very difficult. 

So I want to encourage you to be much more vigilant when people complain about things and find the right words for dealing with problems that are discussed openly.

**Reply by NiklasBr** [reply]
@helloanoop I'm curious about this:

> The majority of the “Golden Edition for Individuals” licenses were from organizations, which was not the design and is not sustainable

Can you perhaps give an example why it is sustainable or not depending on who buys it? As far as I understand the application architecture there is no cloud offering where organisations use much more bandwidth for example. After all, one of the primary benefits I found in Bruno was the distinct lack of such cloud sync and phone-home "features".

**Reply by Stokestack** [reply]
@NiklasBr  Presumably an organization is a bunch of people, and if they only buy _one_ license and hand it out to everyone at the company... that sucks for the Bruno devs. Do they need to explain that further?

@helloanoop Any developer can understand why you went to this model, but not why you guys haven't scrubbed all mention of your previous plans from the site.

**Reply by anhofmann** [reply]
@NiklasBr  
> Can you perhaps give an example why it is sustainable or not [..]?

Running a Company costs money every day. 
Buying a one time license brings in money once. 
This isn't sustainable, it is that easy.

@helloanoop  FWIW I loooooove the pricing of the Jetbrains products. It costs x dollars a year, in the first year. Second year, it is a bit cheaper, from the third year on, the license has a 40% discount. I'm a customer since almost a decade, and it is the only license I pay every year with joy.

**Reply by NiklasBr** [reply]
> Presumably an organization is a bunch of people, and if they only buy one license and hand it out to everyone at the company... that sucks for the Bruno devs. Do they need to explain that further?

I wasn't asking about piracy and breaches of contract @Stokestack, I was asking a question about why a company who buys a license for each of their individual developers is not sustainable, but when private persons are buying the licenses it implicitly is?

> Running a Company costs money every day.

True. How is this different depending on who buys the license?

> Buying a one time license brings in money once.

True. How is this different depending on who buys the license?

> This isn't sustainable, it is that easy.

@anhofmann but the author literally said that it was only unsustainable when a company buys a license for each seat and clearly implied that it was sustainable when private persons buys licenses. I wonder why one is sustainable and the other is not.

**Reply by anhofmann** [reply]
@NiklasBr this is beating around the bush. The question makes no sense, you will not get a useful answer to that. The point is: the bruno people realized that their business model doesn't work. So they changed it. And they communicated not perfectly. This, and the pricing change, makes people disappointed. Every discussion around that point makes no sense. It doesn't matter what they wrote on their blog post and how they worded the facts. 

It only matters if we are able to understand, that bruno will only stay alive, if we give them money. I think this is a fair deal. That the bruno people are breaking promises, doesn't make me happy either, and I'd love to see them to stop this. 

Just to hammer down my point: I know bruno since a year, and I saw that a one time payment will not keep them alive. So I didn't use it.  Now they change to a subscription license, I have hopes that bruno will be alive for the next decades. But if we all just complain and tell the world "bruno betrayed us", then bruno will die. Just like hundreds of other nice products. 

Don't do this. Just pay the people, who work on this.

**Reply by NiklasBr** [reply]
We're not truly in disagreement it seems @anhofmann, no need to be pound the table this hard. :) 

As I mentioned before, I'm here trying to be a positive participant, trying to add both code and otherwise help.

![Screenshot 2025-02-28 at 10 33 20](https://github.com/user-attachments/assets/297abe10-0b9e-4bad-b3c0-47a0524440b8)

I, too, want Bruno to survive. I, too, want to regularly pay Bruno for the value it provides me. 

But the reason I ask these recent questions is that by doing these changes Bruno is degrading the value they are providing. And on top of that, the communication on the topic is still flawed. By asking these question I hope we can get some clarifications which helps everyone and the Bruno folks especially.

As it is right now @helloanoop writes that a company buying e.g. 15 licenses intended for individuals is less sustainable than the same number of licences being bought by individuals. And _that_ is what makes no sense to me. Are those 15 licenses requiring more support overhead? More paperwork? More hardware needed for the Bruno servers? And that is what I am currently focusing on regarding the flawed communication.

**Comment by EvilVir** [comment]
I've just joined this party of people surprised by yet another licensing change. Yes, slowpoke, I know.

In general I think Bruno is slowly moving into same direction that Postman and Insomnia went. Slowly boiled frog approach. First it was "We're the good guys" now it's "Here's yet another stupid subscription plan. And that GE you bought back then? Well that was a mistake, let me take that from you and now again some features are behind paywall.".

This is a bit unfair, to be honest. It already was fishy when GE poped up, but that was still one time fee to support the project. I wouldn't mind if it would be 10-20 times the amount, really. But one time, not that subscription bs.

**Comment by helloanoop** [comment]
Some notes @EvilVir 

> And that GE you bought back then? Well that was a mistake, let me take that from you 

We have not taken anything back. All folks who purchased GE still have access to all the features and will get updates for 2 years and then will be able to use that version for perpetuity

> In general I think Bruno is slowly moving into same direction that Postman and Insomnia went. 

I can understand why you feel that way. I'll let the future be the judge.

> "We're the good guys" now it's "Here's yet another stupid subscription plan.

Agreed. But I still feel we are the good guys. Yes, we have built a business around it. Bootstrapped so far.
90% of the efforts of our team go to maintaining the FOSS version. Its MIT licensed.
Its our gift to eternity.

I must also emphasise that >98% of our users do not purchase a license. Almost everything a dev needs is free. We charge for a few things of course (you need to, otherwise no one would buy things). As a matter of principle we have never taken away any features that were free. We continue to make features free that were paid earlier. 

> But one time, not that subscription bs.

When the project started, it was me alone for 2 years. When writing the [initial vision](https://github.com/usebruno/bruno/discussions/269) - it was me alone.
Over time, the [team](https://www.usebruno.com/team) grew from 1 to 14 today. Its unbearably painstaking to run a company, pay everyones salary, and keep the lights on, and build and maintain the project developers love. One time payments worked when I was alone, it does not scale for a team.

I love building and leading the project.
I also understand that people feel that I sold a vision that I could not keep up (in terms of subscriptions and building a company)

As the projects leader, I decided to have a commercial engine that powers our work as well as takes care of everyone's incentives as the path forward.

**Comment by rubin55** [comment]
@helloanoop Do you plan on doing a similar-to-jetbrains licensing scheme? I.e., you buy at version a for a year, within that year version b and c come out, license expires, you can keep using c, but can't upgrade to d. If one would like to upgrade to say d, e, f, etc, they would buy a license/subscription again. It's a nice balance between keeping customers subscribed for the next new shiny, and also protects customers from argh-suddenly-cant-use-product-because-expire.

An otherwise more expensive perpetual option for say a major version number would also be nice provided those don't change more than once or so a year.

In any case, I just subscribed for a year, I'll give Bruno a try and see how the OAS features work. One annoyance: I couldn't turn off payment recurrence on license.usebruno.com; it seems fixed on AUTOMATIC. Would like MANUAL or somesuch.

**Reply by helloanoop** [reply]
Hey @rubin55 Thanks for the support. Currently, we don't have plans to do perpetual license similar to jetbrains.

One thing we do different from jetbrains is that we continue to release paid features into the free tier over time. Last Oct we made GRPC, Websockets, DevTools free. This Oct we plan to make File Explorer, History and Creating JS files free.

We seem to have arrived at a good business model that keeps most (if not all) stakeholders happy. 

> and see how the OAS features work.

Please note that collection to OAS will always have some fidelity loss. We plan to add OAS sync early next year where you can keep the collection upto date with OAS specs.
